balboa-service-jms/docker/ship.d/run

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -ev
+
+# Parse Jinja2 templates and replace them with configurations.
+/bin/env_parse ${BALBOA_CONFIG}.j2
+/bin/env_parse ${BALBOA_LOG4J}.j2
+
+exec sudo -u socrata java \
+  -Dcom.sun.management.jmxremote.port=40507 \
```

Comment:
You will need to also set com.sun.management.jmxremote.rmi.port (to the same port) to actually be able to talk to it.